test/Twig/Tests/Extension/HumanReadableBytesTest.php

```diff
@@ -0,0 +1,142 @@
+     */
+    protected $humanReadableBytesExtension;
+
+    /**
+     * @var array
+     */
+    protected $humanReadableBytesTestData;
+
+    /**
+     * @var array
+     */
+    protected $humanReadableBytesFormatTestData;
+
+    /**
+     * @var array
+     */
+    protected $humanReadableBytesDefaultExpectedResults;
+
+    /**
+     * @var array
+     */
+    protected $humanReadableBytesFormattedExpectedResults;
+
+    /**
+     * Sets test data.
+     */
+    protected function setTestData()
+    {
+        $this->humanReadableBytesTestData = array(
+            486827,
+            21864948,
+            11378925384,
+            4121781686582782,
+            713,
+        );
+
+        $this->humanReadableBytesFormatTestData = array(
+            array(
+                'bytes' => 486827,
+                'decimalPlaces' => 3,
+                'decimalPoint' => ',',
+                'thousandsSeparator' => ' ',
+                'units' => 'SI',
+            ),
+            array(
+                'bytes' => 21864948,
+                'decimalPlaces' => null,
+                'decimalPoint' => null,
+                'thousandsSeparator' => null,
+                'units' => null,
+            ),
+            array(
+                'bytes' => 11378925384,
+                'decimalPlaces' => 8,
+                'decimalPoint' => ' ',
+                'thousandsSeparator' => '.',
+                'units' => 'SI',
+            ),
+            array(
+                'bytes' => 4121781686582782,
+                'decimalPlaces' => 4,
+                'decimalPoint' => '-',
+                'thousandsSeparator' => '_',
+                'units' => 'SI',
+            ),
+            array(
+                'bytes' => 713,
+                'decimalPlaces' => 0,
+                'decimalPoint' => '.',
+                'thousandsSeparator' => ',',
+                'units' => 'IEC',
+            ),
+        );
+    }
+
+    protected function setExpectedResults()
+    {
+        // defaults are:
+        //  2 decimal places,
+        //  "." decimal point,
+        //  "," thousands separator
+        //  IEC units
+
+        $this->humanReadableBytesDefaultExpectedResults = array(
+            '475.42 KiB',
+            '20.85 MiB',
+            '10.60 GiB',
+            '3,748.74 TiB',
+            '713.00 B',
+        );
+
+        $this->humanReadableBytesFormattedExpectedResults = array(
+            '486,827 KB',
+            '21 MiB',
+            '11 37892538 GB',
+            '4_121-7817 TB',
+            '713 B',
+        );
+    }
+
+    public function setUp()
+    {
+        $this->humanReadableBytesExtension = new Twig_Extensions_Extension_HumanReadableBytes();
+        $this->setTestData();
+        $this->setExpectedResults();
+    }
+
+    /**
+     * Tests defaults.
+     */
+    public function testHumanReadableBytesConversion()
+    {
+        foreach ($this->humanReadableBytesTestData as $key => $value) {
```

Comment:
Writing the test this way does not make sense. Use a PHPUnit data provider instead, which would allow to have all tests running each time instead of having a single test which would pass or fail